package.json

```diff
@@ -7,6 +7,7 @@
   "author": "Tijs Verkoyen <tijs@verkoyen.eu>",
   "license": "MIT",
   "devDependencies": {
+    "@cityssm/simple-password-generator": "^0.6.1",
```

Comment:
Can you use something with a stable release?
This thing was only created 7 days ago...

Reply:
Does it really matter though? It's a random password generator, it's not like it's going to break or something.

Reply:
It does, please use a different library, it is such a common issue that it can't be that hard to use one that is used by somebody else than the creator of this one

Reply:
I agree with Jelmer. 0-releases have the tendency to not being maintained. I understand that this is not a very critical part of Fork, but external packages can have bugs which are a pain in the ass to fix.

Some alternatives:
- https://www.npmjs.com/package/password-generator, seems like its being maintained, has tests.
- https://www.npmjs.com/package/generate-password

Reply:
I updated the plugin